plugin/audit_log_filter/tests/mtr/t/decrypt_rotated_log.inc

```diff
@@ -24,19 +24,39 @@
 --let encryption_iterations = $encryption_iterations
 
 perl;
-  my $dir_name = $ENV{'audit_filter_log_path'} or die "Empty audit_filter_log_path";
-  my $password = $ENV{'encryption_password'} or die "Empty encryption_password";
-  my $iterations = $ENV{'encryption_iterations'} or die "Empty encryption_iterations";
+  my $help_text = qx(openssl enc -help 2>&1);
+  my $openssl_iter_support = 1;
 
-  opendir(my $dh, $dir_name) or die "Can't opendir $dir_name: $!";
+  if ($help_text !~ m/\-iter/) {
+    $openssl_iter_support = 0;
+  }
+
+  open(OUTPUT, ">$ENV{MYSQLTEST_VARDIR}/tmp/audit_decrypt_openssl_status.inc") or die "Cannot open tmp file";
+  print OUTPUT "let \$openssl_iter_support = $openssl_iter_support;\n";
+  close(OUTPUT);
+
+  if ($openssl_iter_support) {
```

Comment:
Just a minor comment. I think the following will be clearer:
```
  if (!$openssl_iter_support) { exit; }
```

Reply:
Updated it